.github/workflows/ci.yaml

```
@@ -68,7 +68,7 @@ jobs:
         run: |
           npm run report
       - name: Coveralls
-        if: ${{ matrix.node-version == 'lts/*' }}
+        if: ${{ matrix.node-version == 'lts/*' && github.repository == 'WFCD/worldstate-emitter' }}
```

actionlint (1.7.9)

property "node-version" is not defined in object type {}

(expression)